ingestion/examples/airflow/airflow.cfg

```diff
@@ -21,7 +21,7 @@ default_timezone = utc
 # ``SequentialExecutor``, ``LocalExecutor``, ``CeleryExecutor``, ``DaskExecutor``,
 # ``KubernetesExecutor``, ``CeleryKubernetesExecutor`` or the
 # full import path to the class when using a custom executor.
-executor = LocalExecutor
+executor = SequentialExecutor
```

Comment:
@ayush-shah any reason to use SequentialExecutor. With SequentialExecutor there could ever be one workflow that can run and others have to wait. If we have multiple pipelines this will be an issue

Reply:
Using SequentialExecutor, we can ingest sample-data & sample-usage first and then elasticsearch indexes and the data related to it from elasticsearch and it's indexes

Reply:
I will revert it to LocalExecutor, and check if I can trigger the dags dependent on other dags